RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py

```diff
@@ -42,6 +42,14 @@
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
Thanks for having a look. 
The goal was to have all the digitization and local reco working for all muon detectors for the upgrade (CSC+DT+GEM+RPC). That is why i was using (and modifying) this file. I will split the pull request in two: 
1)one part dedicated to fixing the RPCDitizer;
2) one to part containing all other changes to have the digi and local reco working for all muon detectors for the upgrade